tools/nl/embeddings/finetuning/finetune.py

```diff
@@ -34,13 +35,19 @@
 
 flags.DEFINE_string('model_name_v2', 'all-MiniLM-L6-v2', 'Model name')
 flags.DEFINE_string('bucket_name_v2', 'datcom-nl-models', 'Storage bucket')
-flags.DEFINE_string('embeddings_size', '',
-                    'Valid values are: "small", "medium"')
+flags.DEFINE_string(
+    'stage', 'alternatives',
```

Comment:
a suggestion: for consistency should we match the stage name to the output file name?  That is refer to the first one as `alternatives` and second one as `final`?

otherwise, the user has to connect that: stage=base produces alternatives, and stage=alternatives produces final.

Reply:
Ok so I changed the params to "final" and "alternatives" where "final" just implies the final (second) stage while "alternatives" implies both the first and second stage. Not sure this is a lot better but hopefully the comments help?

Reply:
Done, thanks